docs/DeveloperGuide.md

```diff
@@ -732,3 +736,11 @@ Timestamp in the status bar is updated.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+# Appendix
+
+## Planned Enhancements
+
+1. Currently, the view panel can only be updated through the `view` command. We plan to make it so that clicking
+on a person in the list panel also updates the view panel to the selected person. The reason is although our application is made for
+CLI users, expecting that clicking on a person updates the view panel seems to be an intuitive and reasonable expected behaviour.
```

Comment:
> Currently, the view panel can solely be updated using the `view` command. However, we are planning on implementing a feature that will allow users to update the view panel by simply clicking on a person in the list panel. This change is driven by our goal to enhance user experience, as it is our belief that such behaviour, while our application primarily caters to CLI users, aligns with intuitive and reasonable expectations.


Extremely small nit.
Might be something personal, but I had to read the last sentence twice to understand the message. Provided an alternative for consideration. Cheers

Reply:
Thanks for the insight! Have rephrased the message to be something similar to your suggestion.